scrooge-generator/src/main/scala/com/twitter/mustache/ScalaObjectHandler.scala

```diff
@@ -0,0 +1,69 @@
+package com.twitter.mustache
```

Comment:
How about we shade it in? We change the package name and make it package-protected, so even if somebody uses scala-extensions, they won't collide?
`package com.twitter.scrooge.mustache.shaded.ScalaObjectHandler`